controllers/glanceapi_controller.go

```diff
@@ -240,6 +244,7 @@ func (r *GlanceAPIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&glancev1.GlanceAPI{}).
 		Owns(&keystonev1.KeystoneEndpoint{}).
+		Owns(&cinderv1.CinderAPI{}).
```

Comment:
ack, thanks @abays.
 I'm actually not sure we need to watch the `Cinder CR` or just double check it exists in the ctlplane. Definitely this line should go away as we don't `own` the resource in glance.
I'll experiment a bit in my next iteration and I'll ping you for a review!

Reply:
I guess since you have a requeue pattern in place for detecting the `Cinder` CR [1], it's perhaps not that important to also watch for them.  I just wonder if their may be situations in which this waiting/requeue might be insufficient.  Perhaps if we notice some delays later on we can take a look at the watch angle again.

[1] https://github.com/openstack-k8s-operators/glance-operator/blob/eaecd68d2867ae7e0fbf7df903ee536df064045b/controllers/glanceapi_controller.go#L498

Reply:
Sure, we can revisit this pattern if it doesn't work properly. The main reason I didn't create a `Watch` function is because we can get the Cinder CR we expect to see in the ctlplane, and requeue (because we propagate the request of the current reconcile loop) in case it's not found. We're not planning to trigger a reconcile loop if anything change in the cinder CR, just because we're working with a backend injected w/ a `configOverride`: it would be tricky keeping Glance and Cinder in sync, and it could cause regressions. This represents the minimal amount of change required to validate the fact that the (human) operator has set Cinder as Glance backend.
Anyway we can always revisit this strategy as long as we find a better pattern.
Thanks @abays for the reviews!